plugins/test/unit/plugins/importers/yum/test_sync.py

```diff
@@ -34,7 +34,7 @@
 
 class BaseSyncTest(unittest.TestCase):
     def setUp(self):
-        self.url = 'http://pulpproject.org/'
+        self.url = 'https://repos.fedorapeople.org/repos/pulp/pulp/demo_repos/zoo/'
```

Comment:
tests are passing without Internet access, it was my mistake - i forgot to mock necessary things in the new tests